terraform/modules/azure/storage.tf

```diff
@@ -0,0 +1,33 @@
+resource "azurerm_storage_account" "tempdata" {
+  name                              = var.tempdata_storage_account_name
+  resource_group_name               = var.resource_group_name
+  location                          = var.region_name
+  account_replication_type          = var.environment_name != "production" ? "LRS" : "RAGRS"
```

Comment:
Can we remove the hardcoded environment name and replace with a storage_account_replication_type variable?

Reply:
Updated accordingly